packages/gatsby-theme-store/src/sdk/seo/product/useMetadata.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * One should use either noindex or canonical, never both
+
+* This deduplicates pages so our pages rank higher in Google
+*/
+import { useLocation } from '@reach/router'
+import { useMemo } from 'react'
+import type { GatsbySeo } from 'gatsby-plugin-next-seo'
+import type { ComponentPropsWithoutRef } from 'react'
+
+import { useCurrency } from '../../localization/useCurrency'
+import { useLocale } from '../../localization/useLocale'
+import type { Product } from './types'
+
+export type Options = {
+  product?: Product
+  siteMetadata: ComponentPropsWithoutRef<typeof GatsbySeo> & { siteUrl: string }
+}
+
+export const useMetadata = (
+  options: Options
+): ComponentPropsWithoutRef<typeof GatsbySeo> => {
+  const language = useLocale()
+  const [currency] = useCurrency()
+  const { pathname, host } = useLocation()
+  const { product, siteMetadata } = options
+  const price = product?.items[0].sellers?.[0].commercialOffer.spotPrice
```

Comment:
```suggestion
  const price = product?.items?.[0].sellers?.[0].commercialOffer?.spotPrice
```

Reply:
Items is required I guess. We were using without the `?` operator on line 33 already, so I think it's safe to use this way. Like this we can remove the extra `if`